src/httpserver/handlers/handler_function_selector.clj

```diff
@@ -0,0 +1,8 @@
+(ns httpserver.handlers.handler_function_selector
```

Comment:
`handler-function-selector` sounds kinda clunky. I feel like this is your application's router, no?